### ISSUE TITLE
Add asset size limit to config

### ISSUE THREAD

**Issue by henrikkorsgaard** [Webstrates/Webstrates#20]
It would be nice if it was possible to add an asset size limit to the configuration file (instead of having to introduce limits in nginx as it is now).

I don't know if the limit should be per webstrate or per asset.

**Comment by raedle** [comment]
I agree with @henrikkorsgaard and I like his suggestion adding the asset size limit per webstrate. Would it be possible to add another `data-` attribute to the `html` document, i.e. `data-max-asset-size="50MB"?

@kba Any concerns with having file size limit configurable in a webstrate? There could still be an overall size limit configured on the server.

@henrikkorsgaard What's the idea behind having a size limit per asset?

**Comment by henrikkorsgaard** [comment]
The suggestion was motivated by me asking if there is a limit currently, mainly because I was wondering if I might hit it with some playing around. I am not sure if there is a need, but perhaps this might be relevant for storage and performance related concerns. 

If this is indeed related to storage and perhaps server performance, then it should, IMO, be a general configuration on the webstrate server and not a nginx configuration (as I understand it is currently). So my initial suggestion was more on moving the configuration from nginx and into webstrate server configuration, to make it visible and configurable as part of setting up a webstrate server.

@raedle a per webstrate limit might be a good idea of large assets have an impact on the performance of the webstrate. 

If there are no performance issue at play (per webstrate or on the server), then my proposal should just be removed. No need to introduce a configuration to address a hypothetical issue.

**Comment by kbadk** [comment]
We're not intentionally introducing a limit to nginx, but rather the other way around. Nginx' default file size limit is 1 MB. I've upped that, so we would be hitting Webstrates' file upload limit before Nginx. By introducing a limit in Webstrates, we wouldn't be making configuration any easier. You'd still have to manually up the limit in the Nginx configuration.

That aside, I agree that defining a very high upperbound in Nginx (maybe 1GB or something ridiculous), and then adding a more practical limit to Webstrates itself would make sense.

I don't see a point of introducing a limit the the total size of assets in a Webstrate. What purpose would that serve? If the intention is to limit how much people will upload, I can't imagine that'll be effective. People could easily just create another webstrate and upload their assets to that once their original webstrate is "full". It seems like a pointless exercise.

Likewise, I'm not sure I understand the purpose of a `data-max-asset-size` attribute. What would this accomplish? If we actually want to make sure that people can't upload files bigger than 50 MB or whtaever it's been defined to, that won't really help at all, because whoever has access to upload a file to a webstrate will also have access to change the attribute.
